src/components/MyAccount/Settings/AccountSettingsButtons.tsx

```diff
@@ -20,12 +20,12 @@ const AccountSettingsButtons = ({
       onClick={() => toggleCurrentlyEditing(true)}
     >
       <Icon name="editorMode" align="left" size="medium" />
-      Edit account Settings
+      Edit account settings
     </Button>
   )
 
   const cancelAndSaveButtons = (
-    <>
+    <ButtonGroup>
```

Comment:
I think this is causing a rendering flicker but I will take a look on the DS side for next week/sprint.

Reply:
BTW, this was fixed in DS 3.1.3